backend/api/views/main.py

```diff
@@ -24,6 +24,30 @@ def get_persons():
     return create_response(data={"persons": persons})
 
 
+# function that is called when you visit /census_response
+@main.route("/census_response_by_datetime", methods=["GET"])
```

Comment:
can we make the url something like `/census_response?datetime="jdskflsdjkl"` bc this endpoint name is tilting me